spotbugs-annotations/build.gradle

```diff
@@ -6,8 +6,12 @@ apply from: "$rootDir/gradle/checkstyle.gradle"
 apply from: "$rootDir/gradle/javadoc.gradle"
 apply from: "$rootDir/gradle/maven.gradle"
 
+repositories {
+	mavenLocal()
+}
+
 dependencies {
-  compile 'com.google.code.findbugs:jsr305:3.0.2'
+  compile 'com.github.spotbugs:jsr-305-annotations:3.0.3-SNAPSHOT'
```

Comment:
better to use `compile project(':jsr-305-annotations')` then we don't need `mavenLocal()` in `repositories`.